pkg/kubectl/polymorphichelpers/helpers.go

```diff
@@ -233,3 +239,60 @@ func SelectorsForObject(object runtime.Object) (namespace string, selector label
 
 	return namespace, selector, nil
 }
+
+// attachablePodForObject returns the pod to which to attach given an object.
+func attachablePodForObject(restClientGetter genericclioptions.RESTClientGetter, object runtime.Object, timeout time.Duration) (*api.Pod, error) {
```

Comment:
make a separate file for these.  makes it obvious which ones are missing tests.  you don't need to add a test, but let's make our debt obvious